jsk_perception/CMakeLists.txt

```diff
@@ -258,6 +260,7 @@ add_library(${PROJECT_NAME} SHARED ${jsk_perception_nodelet_sources}
   slic/slic.cpp
   src/histogram_of_oriented_gradients.cpp
   )
+add_darknet_definitions(${PROJECT_NAME})
```

Comment:
This is for keeping consistency of C structure between darknet library and this nodelet library. darknet is originally C file (not C++), so we must compensate on build time.